app/Helpers/InvoiceHelper.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Helpers;
+
+use App\Enum\OrderStatusEnum;
+use App\Models\Order;
+use Illuminate\Support\Facades\DB;
+
+class InvoiceHelper
+{
+    public static function generateInvoiceNumber(): string
+    {
+        $invoiceNumber = self::getOrderQuantityInCurrentMonth() + 1;
+
+        $now = now();
+        $invoice = $invoiceNumber . '/FV/' . $now->month . '/' . $now->year;
+
+        while (Order::where('invoice_num', '=', $invoice)->exists()) {
+            $invoiceNumber++;
+            $invoice = $invoiceNumber . '/FV/' . $now->month . '/' . $now->year;
+        }
+
+        return $invoice;
+    }
+
+    private static function getOrderQuantityInCurrentMonth(): int
+    {
+        $orderMonthQuantity = DB::table('orders')
```

Comment:
Lepiej jest nie uzywać RAW queries, kiedy jest to możliwe. Na pierwszy rzut oka można to załatwić korzystając z metody count(). 
Nie widzę też zbyt dużej korzyści z `DB::table('orders')` zamast z Eloquent'a. Czyli `App\Models\Order::count()->where...`